tests/integration/sync/it_sync_location_test.py

```diff
@@ -49,31 +33,27 @@ def test_sync_location(db_session):
     )
     db_session.add(company)
     db_session.commit()
-    location = Location(
-        id=100,
-        name="Coconut Bongolive",
-        code="C",
-        company_id=50,
-        country="United States of America",
-        region="West Coast",
-        city="Another city",
-        address="Some address in Another City",
-        longitude=42.2642026,
-        latitude=-172.148146,
-        type="L",
-        status="closed",
-        comment="A location with "
-    )
-    db_session.add(location)
-    db_session.commit()
-    SyncedLocation(
-        location=location,
-        poster_sync=Poster(
-            url="http://localhost:{port}".format(port=port)
-        ),
-        db_session=db_session
-    ).sync()
-    row = Location.query.filter_by(id=location.id).one()
+
+    auth_mock.return_value = 'token'
+    locations_mock.return_value = [{
```

Comment:
@vladarefiev don't we have a location factory for mocks?

Reply:
@amihaiemil I wanted to do it, but saw that there is puzzle for it.